setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='dripconfig',
-    version='0.1.dev1',
+    version='0.1.dev2',
```

Comment:
Why don't we take a bold leap and call it a real point release. Lord knows we have been relying on it.